### ISSUE TITLE
SamplePlayer: support playing a sample in a different pitch

### ISSUE THREAD

**Issue by RobCvdV** [birdofpreyru/react-native-audio#22]
Simply playing back a sound faster or slower, so the pitch and duration scale. No react native plugin seems to support it (at least for both ios and android).

**Comment by birdofpreyru** [comment]
Yeah, that would be a nice feature to have :) I don't really need it myself though, thus either you make a PR, or sponsor me to implement it :sweat_smile:

**Comment by RobCvdV** [comment]
> Yeah, that would be a nice feature to have :) I don't really need it myself though, thus either you make a PR, or sponsor me to implement it 😅

Was considering that It will be hard to get dive into the native code and frameworks. Takes a lot of time.

**Comment by jerryseigle** [comment]
I would support you to do this update. Is there a way you could add the feature so that we can Transpose (Pitch shift) the audio and keep the tempo the same without us getting the chipmunk effect. Also being able to speed or slow down the audio without changing the pitch and without getting chipmunk effect

I will support you for this. Please tell me how

**Comment by birdofpreyru** [comment]
> _I will support you for this. Please tell me how_

I thought,
[![Sponsor](https://raw.githubusercontent.com/birdofpreyru/react-native-audio/master/.README/sponsor.svg)](https://github.com/sponsors/birdofpreyru)
button was quite prominent in the library's README 😃

Though, your ask, if possible to do, will require quite an effort from me, to investigate and implement it. I don't see it happens unless I am paid for a few weeks of work, at California-level of the rate 😅
